hack/lib/golang.sh

```diff
@@ -220,11 +220,6 @@ kubeedge::golang::build_binaries() {
 
 }
 
-
-KUBEEDGE_ALL_CROSS_BINARIES=(
-edgecore
-)
-
 kubeedge::golang::is_cross_build_binary() {
   local key=$1
   for bin in "${KUBEEDGE_ALL_CROSS_BINARIES[@]}" ; do
```

Comment:
here also use this deleted KUBEEDGE_ALL_CROSS_BINARIES variable, also need fix

Reply:
to fix `make crossbuild WHAT=edgecore` error.

Reply:
Done.

Reply:
LGTM test OK